pegasus/sites.v3/code.org/public/ai.haml

```diff
@@ -0,0 +1,178 @@
+---
+title: Learn about Artificial Intelligence (AI) | Code.org
+theme: responsive_full_width
+---
+
+%link{href: '/css/generated/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: "/css/generated/ai-styles.css", rel: "stylesheet"}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
+
+-# FontAwesome icons
+- icon_sparkles = "fa-solid fa-sparkles"
+- icon_lightbulb = "fa-solid fa-lightbulb"
+- icon_magnifying_glass = "fa-solid fa-magnifying-glass"
+- icon_book = "fa-solid fa-book"
+- icon_person_chalkboard = "fa-solid fa-person-chalkboard"
+- icon_video = "fa-solid fa-video"
+
+%section.hero-banner-basic.bg-neutral-dark{style: "display: block !important"}
+  .wrapper
+    .text-wrapper.col-60
+      %h1{style: "color: white; margin-bottom: 0"}=hoc_s(:ai_hero_heading_01)
+      %h2.heading-xxl{style: "color: white"}=hoc_s(:ai_hero_heading_02)
```

Comment:
Is there a reason why the first part is an h1 and the second an h2? It reads a bit funky on VoiceOver due to the different header levels.

Reply:
I did it so we could get an intentional line break on the "It's just code!" line and wanted to avoid two h1s — but maybe it's better to try to break this us w/ css and have the whole string as one string for better translation?

Reply:
@mgc1194 maybe you can speak to what would be better here? These are the two strings in question:
https://github.com/code-dot-org/code-dot-org/blob/d000dfddac4606be40e5e54fc10705e379bf1d1a/pegasus/sites.v3/hourofcode.com/i18n/en.yml#L149-L150

Reply:
I do think for translatability, it'd be better to keep this as one string. But I'm good with it being a follow-up, as I know getting the styling right will be tricky

Reply:
Cool, ty! I made a [ticket](https://codedotorg.atlassian.net/browse/ACQ-534?atlOrigin=eyJpIjoiNzBmNWRkYTZiNTk1NDk0MDhkMGYwYmYzYjg4NjkxZTkiLCJwIjoiaiJ9) to follow up.

Reply:
Great, thanks!